dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -346,6 +347,49 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 						);
 					}
 
+					if (
+						collection.containerPalette === 'Branded' &&
+						renderAds
```

Comment:
we still render labs container even with reject all - double check

Reply:
I thought it might be useful to expand on the current state of ad free and `canRenderAds`.

Thanks to our ad free expert @emma-imber we have in depth documentation here:
https://docs.google.com/document/d/10Mxxta6ypxAduVXJFp19uRoWtALQmAubdKtBwq7AqZ0/edit#heading=h.8g8uwordyell

TLDR:

`isAdFree` is available server side via a Fastly header but originates as a client side cookie, set in the following scenarios:

- the user rejects all or does not give consent to googletag
- the user is a subscriber

On DCR the `canRenderAds` function checks `isAdFree` and `shouldHideAds` (e.g. sensitive article content).

So wrt this PR if the user 'rejects all' I expect the labs container will _not_ be shown.

However since the recent introduction of consentless ads we are planning to decouple 'reject all' from 'isAdFree'.

So in future we will move to a model where 'reject all' users will see a consentless ad experience which includes labs containers. There is no immediate impact to any current labs work as we will handle the migration but just as an fyi.

Reply:
A point that just occurred to me that I thought I should add - at the moment, with the consentless advertising switch on, we unset GU_AF1 if the reason for it was reject all. I suspect at the moment because of this, the labs container will show in a reject all environment. But if we turn the consentless switch back off, reject all would then hide the labs container. I think whether or not the labs container shows might currently be coupled to the consentless advertising switch?

Reply:
Ah good point @emma-imber ! I think in the new consentless world we want the following:

- the user rejects all: Labs should render
- the user is signed in but not a subscriber: Labs should render
- the user is signed in and is a subscriber: Labs should **not** render

And that is the current behaviour with the consentless switch on but we (commercial) will have to think about how we handle a full transition to consentless without the switch.

Either way I think webex should not worry about how `isAdFree` is affected by 'reject all' and we will handle ensuring the correct behaviour is followed?

Reply:
Yep completely agree! I'll add this point to the ticket we have around changing the adfree cookie behaviour when we reject all to make sure this is taken into account.